README.rst

```diff
@@ -178,62 +178,86 @@ GetComponentImages example:
+    image = Image.open('/usr/src/tesseract/testing/phototest.tif')
+    with PyTessBaseAPI() as api:
         api.SetImage(image)
         api.Recognize()
+        
+        it = api.GetIterator()
+        for line in iterate_level(it, RIL.TEXTLINE):
+            text = line.GetUTF8Text(RIL.TEXTLINE)
+            conf = line.Confidence(RIL.TEXTLINE)
+            bbox = line.BoundingBox(RIL.TEXTLINE)
+            bbox = {'x': int(bbox[0]),
+                    'y': int(bbox[1]),
+                    'w': int(bbox[2])-int(bbox[0]),
+                    'h': int(bbox[3])-int(bbox[1])}
+            print(u"Box[{0}]: x={x}, y={y}, w={w}, h={h}, "
+                  "confidence: {1}, text: {2}".format(i, conf, text, **bbox))
 
-        it = api.AnalyseLayout()
-        orientation, direction, order, deskew_angle = it.Orientation()
-        print("Orientation: {:d}".format(orientation))
-        print("WritingDirection: {:d}".format(direction))
-        print("TextlineOrder: {:d}".format(order))
-        print("Deskew angle: {:.4f}".format(deskew_angle))
-
-or more simply with ``OSD_ONLY`` page segmentation mode:
+Layout analysis with orientation and deskewing:
+```````````````````````````````````````````````
 
 .. code:: python
 
+    import math
+    from PIL import Image
     from tesserocr import PyTessBaseAPI, PSM
+    from tesserocr import Orientation, WritingDirection, TextlineOrder
 
-    with PyTessBaseAPI(psm=PSM.OSD_ONLY) as api:
-        api.SetImageFile("/usr/src/tesseract/testing/eurotext.tif")
-
-        os = api.DetectOS()
-        print("Orientation: {orientation}\nOrientation confidence: {oconfidence}\n"
-              "Script: {script}\nScript confidence: {sconfidence}".format(**os))
+    with PyTessBaseAPI(psm=PSM.AUTO) as api:
+        image = Image.open("/usr/src/tesseract/testing/eurotext.tif")
+        api.SetImage(image)
 
-more human-readable info with tesseract 4+ (demonstrates LSTM engine usage):
+        it = api.AnalyseLayout()
+        orientation, direction, order, deskew_angle = it.Orientation()
+        print("Orientation: {}".format(membername(Orientation, orientation)))
+        print("WritingDirection: {}".format(membername(WritingDirection, direction)))
+        print("TextlineOrder: {:d}".format(membername(TextlineOrder, order)))
+        print("Deskew angle: {:.1f}°".format(deskew_angle * 180 / math.pi))
+    
+    def membername(class_, val):
+        """Convert a member value into a member name string."""
+        return next((k for k, v in class_.__dict__.items() if v == val), str(val))
+
+Orientation and script detection legacy model:
+``````````````````````````````````````````````
 
 .. code:: python
 
     from tesserocr import PyTessBaseAPI, PSM, OEM
 
-    with PyTessBaseAPI(psm=PSM.OSD_ONLY, oem=OEM.LSTM_ONLY) as api:
+    with PyTessBaseAPI(psm=PSM.OSD_ONLY, 
+                       oem=OEM.TESSERACT_ONLY, 
+                       lang="osd") as api:
```

Comment:
Believe me, it _does not work_ with LSTMs. The standalone CLI even loads the `osd` model anyway if the user forgot to. On the API, it will _look like_ it works without loading it, because the default model `eng` will get loaded. But there are only symbols from one script in that model, `Latin`, thus no _actual_ script detection would happen. (The "signal" would always be strong, because no competing scripts are loaded, like in `osd`. OSD is very special, because it does _not_ use multiple models, but needs a single model with multiple scripts – for which `osd` is of course the most versatile, but also `frk` contains both `Latin` and `Fraktur`, `hin` contains some `Latin` as well etc.)